client/src/frontend/components/entity/Collection/patterns/Journals.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import PropTypes from "prop-types";
+import get from "lodash/get";
+import { useTranslation } from "react-i18next";
+import isEmpty from "lodash/isEmpty";
+import EntityThumbnail from "global/components/atomic/EntityThumbnail";
+import ThumbnailGrid from "global/components/entity/ThumbnailGrid";
+import EntityCollection from "../EntityCollection";
+import * as shapes from "../shapes";
+
+function JournalsEntityCollection({
+  journals,
+  meta,
+  filterProps,
+  paginationProps,
+  ...passThroughProps
+}) {
+  const { t } = useTranslation();
+
+  const showFilters = !isEmpty(meta) && !isEmpty(filterProps);
+  return (
+    <EntityCollection
+      title={t("titles.journals_all")}
```

Comment:
@1aurend @jen-castiron I wonder if it makes sense to get rid of the "All" in the titles for these pages. Wouldn't it be fine to just say "Projects", "Project Collections", "Journals"?

Reply:
Also sounds good to me. We'll just have to sort out the mobile labels and whether we want consistency in admin:
<img width="442" height="377" alt="Screenshot 2025-11-11 at 3 45 20 PM" src="https://github.com/user-attachments/assets/3f0a2efe-303d-454a-abd6-254f8856d052" />
<img width="224" height="30" alt="Screenshot 2025-11-11 at 3 45 36 PM" src="https://github.com/user-attachments/assets/b412bbf9-a91e-4fc2-b977-62ab3d7609ce" />
<img width="249" height="240" alt="Screenshot 2025-11-11 at 3 47 19 PM" src="https://github.com/user-attachments/assets/974dc1da-e052-4859-b5cb-542242ce9c7c" />

Reply:
Ah, thanks for raising this. The "All" makes sense in those contexts. Still, what if we keep the mobile nav title but just use a new page title? That's how the backend routes have it.